src/Mailer/Email.php

```diff
@@ -1830,6 +1841,29 @@ public function message($type = null)
     }
 
     /**
+     * Sets priority.
+     *
+     * @param int|null $priority 1 (highest) to 5 (lowest)
+     * @return $this
+     */
+    public function setPriority($priority)
+    {
+        $this->_priority = $priority;
```

Comment:
In general I agree with you, for all kind of methods (and soon with php7.1 we will be finally able to assert it with type strictness).

But in this case it can even be a string, e.g. check sources like https://www.example-code.com/asp/emailObject_setPriority.asp
So adding a comment afterwards for human readability seems to be fine and is not supposed to affect the value.

In short: In this specific method we do not need any validation IMO.